resources/com/github/coreos/buildconfig.json

```diff
@@ -0,0 +1,163 @@
+    "name": "${NAME}",
+    "annotations": {
+      "openshift.io/display-name": "Buildconfig for ${NAME}",
+      "description": "${NAME}",
+      "template.openshift.io/bindable": "false"
+    }
+  },
+  "labels": {
+      "template": "${NAME}",
+      "app": "${NAME}"
+  },
+  "objects": [
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}",
+        "annotations": {
+          "description": "Keeps track of changes in the application image"
+        },
+	"ownerReferences": [
+          {
+           "apiVersion": "batch/v1",
+           "kind": "Job",
+           "name": "${OWNERNAME}",
+           "uid": "${UID}",
+           "controller": true
+          }
+        ]
+      },
+      "spec": {
+        "failedBuildHistoryLimit": 1,
+        "successfulBuildsHistoryLimit": 1,
+        "lookupPolicy": {
+          "local": true
+        }
+      }
+    },
+    {
+      "kind": "BuildConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}",
+        "annotations": {
+          "description": "Defines how to build the application",
+          "template.alpha.openshift.io/wait-for-ready": "true"
+        },
+        "ownerReferences": [
+          {
+           "apiVersion": "batch/v1",
+           "kind": "Job",
+           "name": "${OWNERNAME}",
+           "uid": "${UID}",
+           "controller": true
+          }
+        ]
+      },
+      "spec": {
+        "source": {
+          "type": "Git",
+          "git": {
+            "uri": "${SOURCE_REPOSITORY_URL}",
+            "ref": "${SOURCE_REPOSITORY_REF}"
+          }
+        },
+        "strategy": {
+          "dockerStrategy": {
+            "dockerfilePath": "${DOCKERFILE}",
+            "env": [
+              {
+                "name": "GIT_URL",
+                "value": "${SOURCE_REPOSITORY_URL}"
+              },
+              {
+                "name": "GIT_REF",
+                "value": "${SOURCE_REPOSITORY_REF}"
+              },
+              {
+                "name": "OPENSHIFT_GIT_HACK",
+                "value": "TRUE"
+              }
+            ],
+            "from": {
+              "kind": "ImageStreamTag",
+              "namespace": "${NAMESPACE}",
+              "name": "fedora:33"
+            }
+          },
+        "dockerfilePath": "${DOCKERFILE}"
+        },
+        "output": {
+          "to": {
+            "kind": "ImageStreamTag",
+            "name": "${NAME}:${IMAGETAG}"
+          }
+        }
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the frontend objects defined in this template.",
+      "required": true,
+      "value": "cosa-image"
+    },
+    {
+      "name": "NAMESPACE",
+      "displayName": "Namespace",
+      "description": "The OpenShift Namespace where the ImageStream resides.",
+      "required": true,
+      "value": "fedora-coreos"
```

Comment:
This should default to `coreos-ci` instead, since that's the name of the project where all the upstream CI runs.